test/norm_num.lean

```diff
@@ -71,18 +71,18 @@ by norm_num at *; exact h
 -- example : (2 * 12868 + 25705) * 11621 ^ 2 ≤ 23235 ^ 2 * 12868 := by norm_num1
 
 -- example (x : ℕ) : ℕ := by
---   let n : ℕ := by apply_normed (2^32 - 71);
+--   let n : ℕ := by apply_normed (2^32 - 71)
 --   exact n
 
-example (a : ℚ) (h : 3⁻¹ * a = a) : true := by
+example (a : ℚ) (h : 3⁻¹ * a = a) : True := by
   norm_num at h
   guard_hyp h : 1 / 3 * a = a
   trivial
 
--- example (h : (5 : ℤ) ∣ 2) : false := by norm_num at h
--- example (h : false) : false := by norm_num at h
-example : true := by norm_num
-example : true ∧ true := by norm_num
+example (h : (5 : ℤ) ∣ 2) : False := by norm_num at h
```

Comment:
This one is not being done by `norm_num`; there's no extension for `∣` yet.  Instead it it's being done by the `simp` call built into `norm_num`; in fact `simp only at h` also does it.

It's actually pretty weird behaviour -- my best guess is that `simp only` does some decidability checking.  I'll post on Zulip.  But for now can you please post a note similar to the note at line 23
```text
 -- [fixme] simp doing the work not norm_num
```

Reply:
https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/.60simp.20only.60.20does.20decidable.20checks.3F

Reply:
As I said on the zulip (but to repeat here for posterity): I don't like having different standards for tests than failure/success. If this test doesn't test what it should, we should make a better test. Beyond that, the behavior of `norm_num` to use `simp` including the bells and whistles is part of the intended design, so I don't think we need to treat this as a failing test at all.